olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -827,7 +830,7 @@ spec:
                 serviceAccountName: knative-operator
                 containers:
                   - name: knative-operator
-                    image: registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-rhel8-operator@sha256:bacaa4e6cb25927c214c6d51ffc1bee0e3a0f0b70877f9d1eee8c6d739ad8ba4
+                    image: registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-operator@sha256:25ee3bcdf8a904965de9b79dcfd542336670fd9400b4275457221dfe9d59c395
```

Comment:
```suggestion
                    image: registry.redhat.io/openshift-serverless-1/serverless-openshift-kn-rhel8-operator@sha256:bacaa4e6cb25927c214c6d51ffc1bee0e3a0f0b70877f9d1eee8c6d739ad8ba4
```

Reply:
Wondering, why this didn't come through the merge from main, as this is in main: 
https://github.com/openshift-knative/serverless-operator/blob/b11e7405664a378c1ebcffd46e14d8d181997860/olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml#L830